src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/Utilities.cs

```diff
@@ -124,7 +124,7 @@ internal static object GetPrimitiveDefaultValue(Type type)
             switch (type.GetTypeCode())
             {
                 case TypeCode.Boolean:
-                    result = ScriptingRuntimeHelpers.False;
+                    result = False;
```

Comment:
I'll check how many places we use it and rename to `BoxedFalse`. If there's too many places where it gets changed, I'll just revert to using the type name.